package.json

```diff
@@ -108,6 +124,18 @@
 			{
 				"command": "file-switch.relatedFiles",
 				"title": "File Switch: Related Files"
+			},
+			{
+				"command": "path-copy.copyrelative",
+				"title": "Copy Relative"
```

Comment:
Prefix `File Ops:` before every command. And absolute path is being copied not relative.
so - `File Ops: Copy path` is fine